fenix/app/src/main/java/org/mozilla/fenix/settings/SecretSettingsFragment.kt

```diff
@@ -86,6 +97,18 @@ class SecretSettingsFragment : PreferenceFragmentCompat() {
             }
         }
 
```

Comment:
Might be nice to include a small comment similar to the block below? e.g.  "We only want to enable the debug drawer for Nightly or Debug builds because..."